agents/exec/src/main/resources/org.eclipse.che.exec.script.sh

```diff
@@ -23,7 +32,7 @@ if [ ${CURL_INSTALLED} = false ] && [ ${WGET_INSTALLED} = false ]; then
   CURL_INSTALLED=true
 fi
 
-test "$(id -u)" = 0 || SUDO="sudo -E"
+is_current_user_root && is_current_user_sudoer || SUDO="sudo -E"
```

Comment:
@garagatyi if current user is root this code is not setting SUDO variable. Your scenario should work just fine. I've just tested with:

```bash
unset SUDO
is_current_user_root() {
    return 0
}

is_current_user_sudoer() {
    return 0
}

is_current_user_root && is_current_user_sudoer || SUDO="sudo -E"
echo SUDO=${SUDO}

# output sould be: 
# SUDO=
```

But since it doesn't look easy to understand to me neither I've extracted it to a function and made it more readable and covered by the tests:

```bash
set_sudo_command() {
    if is_current_user_sudoer && ! is_current_user_root; then SUDO="sudo -E"; else unset SUDO; fi
}
```